yarn.lock

```diff
@@ -8,22 +8,22 @@
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
```

Comment:
I believe yarn.lock should not be part of any commit. Should we add it to .gitignore?

Reply:
@mbeckerle, yarn.lock should be checked in.  See references below for why:

https://classic.yarnpkg.com/lang/en/docs/yarn-lock/
https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for